dist/index.html

```diff
@@ -1 +1,41 @@
-<!doctype html><html lang="en"><head><meta charset="UTF-8"><meta name="viewport" content="width=device-width,initial-scale=1"><title>Webpack Exercise</title><script defer="defer" src="index.bundle.js"></script></head><body></body></html>
\ No newline at end of file
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@4.3.1/dist/css/bootstrap.min.css" integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
+    <title>Leaderboard</title>
+<script defer src="runtime.bundle.js"></script><script defer src="index.bundle.js"></script></head>
+<body class="p-5">
+    <div class="container">
+        <h1 class="fs-1">Leaderboard</h1>
+        <div class="row">
+            <div class="col pt-5">
+                <div class="row text-center mb-4">
+                    <div class="col">
+                        <h3 class="fs-3">Recent scores</h3>
+                    </div>
+                    <div class="col m-auto"><button id="refresh-btn" class="btn btn-primary" type="button">Refresh</button></div>
```

Comment:
- [ Required ] I think it would be better if the text `Recent scores` is aligned to the left and is on the same line with the scores `table` as illustrated in the project requirements wireframe. 

- [ Required ] I think it would be better if the Refresh `button` is aligned to the right and is on the same line with the right border of the scores `table` as illustrated in the project requirements wireframe. 

## Project requirements
<img src="https://github.com/microverseinc/curriculum-javascript/blob/main/leaderboard/images/leaderboard_wireframe.png?raw=true" width="500px" />

## Your design
<img src="https://user-images.githubusercontent.com/16392046/225993444-82c3de1b-f416-4b5c-9d2d-eba5d6151386.png" width="500px" />